### ISSUE TITLE
Split this repo by platform

### ISSUE THREAD

**Issue by rustema** [rustema/peggy-meter#36]
Android and iOS implementations each deserve their own repository.
Quoting @wahibhaq :
> Wanted to open discussion on something about Github repo structure. IMHO Android and iOS projects better not be in same repo. Eventually both will grow and when a new member has to checkout the repo, it’ll be too big for him/her even when he/she is only interested in one platform. Also, separate repos translates to separate management of `Issues` and keeping things more clear and streamlined.

**Comment by wahibhaq** [comment]
Something to add further, it makes a lot of sense to have a **Modular** app structure where you can extract commonly used functionality in `modules` which makes it reusable. Again, not a compulsion but a good practice when needed.

**Comment by rustema** [comment]
@wahibhaq agreed. Do you know a good example app which implements this structure so we can follow?

**Comment by wahibhaq** [comment]
@rustema this might be helpful https://github.com/rakawestu/modular-android-example
